CHANGES

```diff
@@ -4,6 +4,7 @@
 - API Change: Exposed method ControllerManager#clearListeners on Controllers class
 - API Change: Net#openURI now returns a boolean to indicate whether the uri was actually opened.
 - API Change: DefaultShader now always combines material and environment attributes
+- API Change: Added ShapeRenderer constructor to pass a custom shader program to ImmediateModeRenderer20.
```

Comment:
Everybody uses "API change", but probably would be better to use that (or something else) for changes that affect existing code.